ci/aks/aks-cluster.lokocfg.envsubst

```diff
@@ -116,3 +116,7 @@ component "httpbin" {
 
   certmanager_cluster_issuer = "letsencrypt-staging"
 }
+
+component "experimental-istio-operator" {
```

Comment:
This does not enable the tests, just adds istio operator to be deployed on CI systems. Either we should merge it with commit adding tests, or change the title of the commit.